### DISCUSSION TITLE
Required unit of equivalent Young's modulus

### DISCUSSION THREAD

**Discussion by sitangshugk95** [ContactEngineering/ContactMechanics#86] [Ideas] [answered]
I noticed that it is mentioned in a comment in the [Hardwall](https://github.com/ContactEngineering/ContactMechanics/blob/master/examples/Hardwall_Simulation.ipynb) example code that the unit of equivalent Young's modulus needs to be in MPa. 

It might be a good idea to also explicitly specify that in the source [code](https://contactengineering.github.io/ContactMechanics/_modules/ContactMechanics/FFTElasticHalfSpace.html#FreeFFTElasticHalfSpace)?

**Comment by sannant** [comment, marked as answer]
No the unit is MPa just because I give a concrete example for steel of steel against steel contacts.

Maybe related: 

https://github.com/ContactEngineering/TopoBank/discussions/888
